examples/src/main/java/com/mapbox/navigation/examples/util/RouteOptionsEx.kt

```diff
@@ -0,0 +1,14 @@
+@file:JvmName("RouteOptionsEx")
+package com.mapbox.navigation.examples.util
+
+import android.content.Context
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.navigation.base.extensions.applyDefaultOptions
+import com.mapbox.navigation.base.extensions.applyLocationAndVoiceUnit
+import com.mapbox.navigation.base.extensions.applyRecommendedOptions
+
+fun RouteOptions.Builder.applyAllOptions(context: Context) = apply {
```

Comment:
Should we also add this extension function into `RouteOptionsEx`?

Reply:
Opted to remove it instead.